DownloadAlbum.user.js

```diff
@@ -12,9 +12,9 @@
     'use strict'
     
     // Set preferred download format here
-    var format = "MP3 320"
+    var format = "MP3 V0" 
     // Whether the download tab should automatically be closed after the download has been started
-    var closeAfterDownload = true
+    var closeAfterDownload = false //some weirdo shit happens when this is true. Will need to investigate more
```

Comment:
What's the weirdo shit?

Reply:
Not sure, I was missing downloads when close after download was true but when I left it off it seemed to work fine. I'm not confident on what the issue was so I just noted for myself the behavior seemed off so if I wanted to investigate it I could